<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgLog</class>
 <widget class="QWidget" name="wdgLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Log de myquotesd</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabEstado">
        <attribute name="title">
         <string>Estado de myquotesd</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="myQTableWidget" name="table">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Fuente</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Proceso</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Último cambio estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Número consultas internet</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblLastUpdateStatus">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 128, 0);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabRegistro">
        <attribute name="title">
         <string>Registro de myquotesd</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Introduzca un texto para realizar un filtro</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txt"/>
             </item>
             <item>
              <widget class="QPushButton" name="cmd">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/images/document-preview.png</normaloff>:/images/document-preview.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="lst"/>
           </item>
           <item>
            <widget class="QLabel" name="lblFound">
             <property name="text">
              <string>Registros encontrados</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblLastUpdateLog">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 128, 0);</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txt</tabstop>
  <tabstop>cmd</tabstop>
 </tabstops>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
